### PR TITLE
Improvement/env vars

### PR DESCRIPTION
**Pull request by duhnnie**
fix the issue at trying to read env vars from Heroku deployment conf.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,3 +1,3 @@
 production=false
-REACT_APP_TOKEN_SERVER=https://github-user-stalker-be.herokuapp.com/access_token
-REACT_APP_CLIENT_ID=f1f22fa3391ef25e7d80
\ No newline at end of file
+REACT_APP_TOKEN_SERVER=
+REACT_APP_CLIENT_ID=
```
Comment (duhnnie) [inline .env]: Here the env var values were removed, they will be set up manually at dev and prod. 

The PR includes a ReadMe.md file with the guide about how to set this values.

Comment (ougarte) [reply .env]: I think, you could add a new empty line at the end of file...

File: src/services/resources/auth.js [hunk 2]
```diff
@@ -63,7 +63,7 @@ export function isLogged () {
 export function getLoggedUserInformation(accessToken) {
   return axios({
     method: 'get',
-    url: API_URL + '/user',
+    url: API_HOST + '/user',
```
Comment (ougarte) [inline src/services/resources/auth.js]: I think, you could use string templates...

File: .env (hunk 1 above)
Comment (duhnnie) [reply .env]: Done!

File: src/services/resources/auth.js (hunk 2 above)
Comment (duhnnie) [reply src/services/resources/auth.js]: Done!

**Pull request merged**